src/components/FlawForm.vue

```diff
@@ -487,10 +488,11 @@ const expandFocusedComponent = (parent_uuid: string) => {
       <div v-if="mode === 'edit'" class="border-top osim-flaw-form-section">
         <FlawComments
           :comments="flaw.comments"
+          :taskKey="flaw.task_key"
           :error="errors.comments"
           :isSaving="isSaving"
           @comment:addPublicComment="addPublicComment"
-          @refresh:flaw="emit('refresh:flaw')"
+          @disableForm="(value) => formDisabled = value"
```

Comment:
I think here you could just use`v-model:isSaving` and avoid having to introduce another piece of state and emitting

Reply:
I tested out the change and it's not significant, and I don't feel strongly